language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/createvar/CreateVariableWithTypeCodeAction.java

```diff
@@ -224,16 +224,21 @@ private String getTypeName(TypeSymbol symbol, CodeActionContext context) {
         Optional<ModuleSymbol> module = symbol.getModule();
         if (module.isPresent()) {
             String fqPrefix = "";
-            if (!(ProjectConstants.ANON_ORG.equals(module.get().id().orgName()))) {
-                ModuleID id = module.get().id();
+            ModuleID id = module.get().id();
+            if (!(ProjectConstants.ANON_ORG.equals(id.orgName())) && !isLangAnnotationModule(id)) {
```

Comment:
```suggestion
            if (!ProjectConstants.ANON_ORG.equals(id.orgName()) && !isLangAnnotationModule(id)) {
```
Shall we remove the additional brackets?

Reply:
Done